DeepCat.js

```diff
@@ -473,8 +473,18 @@
 		return false;
 	}
 
+	function enableImeAndSuggestions() {
+		$( '.suggestions' ).css( 'z-index', 'auto' );
+		$( '.imeselector' ).css( 'z-index', 'auto' );
+	}
+
+	function disableImeAndSuggestions() {
```

Comment:
Could be merged into a single `enableImeAndSuggestions( enabled )`, followed by `enabled === false ? -1 : 'auto'`.